wsgi.py

```diff
@@ -0,0 +1 @@
+from app import app
```

Comment:
Why do we need this file?

Reply:
The flask cli will automatically look for `wsgi.py` so if it exists we don't need to set the startup file via a environment variable. That makes using the cli a bit more convenient so I wanted to put it back in.